cfme/services/workloads.py

```diff
@@ -2,28 +2,79 @@
 """ A model of Workloads page in CFME
 """
 from functools import partial
-from cfme.web_ui import accordion, menu
 
-vm_instances_tree = partial(accordion.tree, "VMs & Instances")
-templates_images_tree = partial(accordion.tree, "Templates & Images")
+from navmazing import NavigateToSibling, NavigateToAttribute  # NOQA
+from cfme.exceptions import DestinationNotFound
+from cfme.web_ui import accordion, match_location, toolbar
+from utils.appliance import Navigatable
+from utils.appliance.implementations.ui import navigator, CFMENavigateStep
 
+vm_instances_tree = partial(accordion.tree, "VMs & Instances", 'All VMs & Instances')
+templates_images_tree = partial(accordion.tree, "Templates & Images", 'All Templates & Images')
 
-@menu.extend_nav
-class services_workloads:
-    def service_vms_instances(_):
-        vm_instances_tree("All VMs & Instances")
+QUADICON_TITLE_LOCATOR = ("//div[@id='quadicon']/../../../tr/td/a[contains(@href,'vm_infra/x_show')"
+                          " or contains(@href, '/show/')]")
 
-    def service_vms_instances_filter_folder(ctx):
-        vm_instances_tree("All VMs & Instances", ctx["folder_name"])
+# Only this buttons present. may be used in future
+cfg_btn = partial(toolbar.select, 'Configuration')
+pol_btn = partial(toolbar.select, 'Policy')
+lcl_btn = partial(toolbar.select, 'Lifecycle')
 
-    def service_vms_instances_filter(ctx):
-        vm_instances_tree("All VMs & Instances", ctx["folder_name"], ctx["filter_name"])
+match_page = partial(match_location, controller='vm_or_template')
```

Comment:
Include `title=''` argument as well, matches against xpath:/html/head/title with the 'CFME: ' stripped.